common/app/views/fragments/atoms/quiz.scala.html

```diff
@@ -29,62 +30,108 @@
         }">
         <legend class="atom-quiz__question-text">@question.text</legend>
 
-        @question.answers.map( answer => {
-            if(showResults) {
-                val isCorrect = maybeResults.flatMap(_.isCorrectAnswer(question, answer))
-                renderAnswerWithResponse(question, answer, isCorrect, maybeUserAnswer)
-            } else {
-                renderAnswer(question, answer, field)
-            }
+        @question.imageMedia.map { image =>
+            @fragments.img(image, widthsByBreakpoint = BodyMedia.inline)
+        }
+
+        <div class="atom-quiz__answers @{if(hasImages(question)) "atom-quiz__answers-images"}">
```

Comment:
Would this not be:
`<div class="atom-quiz__answers @{if(hasImages(question)) "atom-quiz__answers--images"}">`